pkg/controller/controller.go

```diff
@@ -400,23 +406,28 @@ func (c *Controller) syncHandler(key string) error {
 			}
 		}
 
+		klog.Info("AM-2 Config")
```

Comment:
These log lines do not provide any useful information. We have to improve these log lines

Reply:
I'll remove them.